FiveCalls/FiveCalls/IssueDone.swift

```diff
@@ -140,25 +232,37 @@ struct CountingView: View {
         } else if count < 4500000 {
             return 5000000
         }
-        
+
         return 0
     }
 }
 
 #Preview {
-    IssueDone(issue: .basicPreviewIssue)
+    let previewState = {
+        let state = AppState()
+            state.issues = [
+                Issue.basicPreviewIssue,
+                Issue.multilinePreviewIssue
+            ]
+        state.contacts = [.housePreviewContact, .senatePreviewContact1, .senatePreviewContact2]
+        state.issueCompletion[Issue.basicPreviewIssue.id] = ["\(Contact.housePreviewContact.id)-voicemail","\(Contact.senatePreviewContact1.id)-contact"]
+        return state
+    }()
+
+    return IssueDone(issue: .basicPreviewIssue)
+        .environmentObject(Store(state: previewState, middlewares: [appMiddleware()]))
 }
 
-struct IssueNavModel {
-    let issue: Issue
+struct IssueDoneNavModel {
+    var issue: Issue
```

Comment:
is there a reason to `var` this? I can't think of a reason to change the nav model after we've created and pushed it onto the nav stack

Reply:
Nope, it was a change that I didn't fully revert. Fixed now.